tutorials/viewports/multiple_resolutions.rst

```diff
@@ -232,21 +238,34 @@ To configure stretching at runtime from a script, use the
 Reducing aliasing on downsampling
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If the game has a very high base resolution (say 4k), aliasing might appear
-when downsampling to something considerably lower like 720. This can be
-detected and reduced by forcing to shrink all images by 2 upon load (this is
-very fast). This can be achieved by calling
+If the game has a very high base resolution (e.g. 3840×2160), aliasing might
+appear when downsampling to something considerably lower like 1280×720.
+Aliasing can be made less visible by shrinking all images by a factor of 2
+upon loading. This can be done by calling the method below before
+the game data is loaded:
 
 ::
 
-    VisualServer.texture_set_shrink_all_x2_on_set_data(true) 
-
+    VisualServer.texture_set_shrink_all_x2_on_set_data(true)
 
-Before most game data is loaded.
 
 Handling aspect ratios
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Once scaling for different resolutions is accounted for, just make sure that your *user interface*
-also scales for different aspect ratios. This can be easily done using :ref:`anchors <doc_size_and_anchors>` 
-and/or :ref:`containers <doc_gui_containers>`.
+Once scaling for different resolutions is accounted for, just make sure that
+your *user interface* also scales for different aspect ratios. This can be
+easily done using :ref:`anchors <doc_size_and_anchors>` and/or :ref:`containers
+<doc_gui_containers>`.
+
+Field of view scaling
+^^^^^^^^^^^^^^^^^^^^^
+
+The 3D Camera node's **Keep Aspect** property defaults to the **Keep Height**
+scaling mode (also called *Hor+*). This is usually the best value for desktop
+games and mobile games in landscape mode, as widescreen displays will
+automatically use a wider field of view.
+
+However, if your 3D game is intended to be played in portrait mode, it may make
+more sense to use **Keep Height** instead (also called *Vert-*). This way,
```

Comment:
Should this one be **Keep Width**? Or maybe the one above?

Reply:
Good catch :smiley:

The default is **Keep Height**, so we're explicitly setting it to **Keep Width** here.